app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx

```diff
@@ -174,6 +174,10 @@ export const connectedStateCellIcon = css`
 
 export const connectedStateCellEvent = css`
 	margin-left: ${token('space.100')};
+    max-width: 400px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
You have an example of what this looks like when the contents hits the edge?

Reply:
yup i will put up a pic